docs/styling-engines.md

```diff
@@ -0,0 +1,25 @@
+# 🕴️ Styling engines
+
+### Using UIToolkit in external windows 🪟
```

Comment:
Do you only have to add the CacheProvider around Dropdown and Tooltip?

This would mean that it would be a breaking change if any other component in UI Toolkit starts to use `@emotion`, because everyone that implemented according to this documentation would have to update.

Reply:
It would be better to have a general statement that all usage of UI Toolkit needs to be wrapped in a CacheProvider. Then we would be safe to continue to use `@emotion ` in all components.

However, if we decide to change to use a different library instead, this would be a breaking change for all users. It would also be a breaking change if we upgrade `@emotion` to a new major version, since all users would have to follow.

Since it is only these two components that use `@emotion`, I think it would be better to completely remove the use of `@emotion` and completely avoid the implications for the user of the library.

Reply:
cc @symphony-adnane

Reply:
`react-select`, which is the structure of Dropdown uses @emotion, and `Popper` which position the `Tooltip` uses @emotion so it's quite the procedure to get them out of the library. Besides, if we replace it with anything else that uses jss for example @material-ui, we'd need an injection point.

Reply:
Technically you only need it around the `Dropdown` and `Tooltip` but yes I think there should be a general statement for the entire library.